### PR TITLE
Added track.js img file

### LINKED ISSUE #78: Add TrackJS logo to the sponsor me section
Just like #72 - but blocked on #47

### PR DESCRIPTION
**Pull request by thomasjosephgreco**
Let me know your thoughts :smile: 

Closes #78

### CODE REVIEW

**Comment by shakeelmohamed** [comment]
Can we get a more rectangular logo? Maybe crop this one to match the dimensions of the browserstack logo?

PS: I won't merge this until we close #47 - which is pretty much blocked on me :snail:

**Comment by thomasjosephgreco** [comment]
@shakeelmohamed yeah not a problem. And I did not even see #47 but I don't have a problem helping out there :smile:

**Comment by shakeelmohamed** [comment]
Great, thank you!

File: index.html [hunk 1]
```diff
@@ -92,6 +92,7 @@
         <br>
         <h3>Sponsored by:</h3>
         <p><a href="https://www.browserstack.com/" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack logo"></a></p>
+        <p><a href="https://trackjs.com" target="_blank"><img src="img/trackjs.png" alt="Browserstack logo"></a></p>
```
Comment (shakeelmohamed) [inline index.html]: We should update the alt text here as well

**Comment by thomasjosephgreco** [comment]
@shakeelmohamed Ok, so i've added the image along with TrackJS support in the header. If i am correct, they gave me an open-source account so I believe we can utilize it free of charge. 

I also took the liberty to create bit.ly links for each of them, in case you ever wanted to track the clicks and stuff like that. Let me know if you have any questions :)

**Comment by thomasjosephgreco** [comment]
@shakeelmohamed Oh, I didn't want to forget to say that I created a .bowerrc file. When I would try to install TrackJS using bower, it wasn't showing up in the bower_components folder so I just specified it.

File: index.html [hunk 2]
```diff
@@ -91,7 +92,8 @@
         <!-- Sponsor links -->
         <br>
         <h3>Sponsored by:</h3>
-        <p><a href="https://www.browserstack.com/" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack logo"></a></p>
+        <p><a href="http://bit.ly/1m2uotc" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack logo"></a></p>
+        <p><a href="http://bit.ly/1RyNpA0" target="_blank"><img src="img/trackjs.png" alt="Browserstack logo"></a></p>
```
Comment (shakeelmohamed) [inline index.html]: The alt text here needs to be `alt="TrackJS logo"`

File: index.html [hunk 3]
```diff
@@ -16,9 +16,10 @@
     <link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.min.css">
     <link rel="stylesheet" href="bower_components/seiyria-bootstrap-slider/dist/css/bootstrap-slider.min.css">
     <link rel="stylesheet" href="css/styles.css">
-  
     <script src="bower_components/jquery/dist/jquery.min.js"></script>
     <script src="bower_components/seiyria-bootstrap-slider/dist/bootstrap-slider.min.js"></script>
+    <script type="text/javascript">window._trackJs = { token: 'f58d0b7c4b4248939fddc582f9e55c9d' };</script>
```
Comment (shakeelmohamed) [inline index.html]: Can we move this line of JavaScript to `everything.js`?
I'm wondering if it needs to be the first line in the file, or inside `$(function() { ...` around line 361

File: .bowerrc [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (shakeelmohamed) [inline .bowerrc]: Since this is the default for directory, we shouldn't have to specify it. Which version of bower are you using?

**Comment by shakeelmohamed** [comment]
Thanks @thomasjosephgreco: I actually don't think we need to worry about the clicks for the "sponsored by links" - so I'd prefer not having the bit.ly links.

Overall this looks fine just some minor things, I send you a private message on Gitter. Let's chat a bit before we merge this in.

File: index.html [hunk 5]
```diff
@@ -1,27 +1,35 @@
...
+    </script>
+    <script src="bower_components/trackjs/tracker.js"></script>
     <script src="bower_components/jquery/dist/jquery.min.js"></script>
     <script src="bower_components/seiyria-bootstrap-slider/dist/bootstrap-slider.min.js"></script>
+    <script type="text/javascript">
+    ;
+    </script>
     <script src="https://www.youtube.com/iframe_api"></script>
     <script src="js/everything.js"></script>
+    <script src="bower_components/trackjs/tracker.js"></script>
```
Comment (shakeelmohamed) [inline index.html]: This is a duplicate line, see L#22

File: index.html [hunk 6]
```diff
@@ -1,27 +1,35 @@
...
-  
+    <script type="text/javascript">
+    window._trackJs = {
+        token: "f58d0b7c4b4248939fddc582f9e55c9d"
+    };
+    </script>
+    <script src="bower_components/trackjs/tracker.js"></script>
     <script src="bower_components/jquery/dist/jquery.min.js"></script>
     <script src="bower_components/seiyria-bootstrap-slider/dist/bootstrap-slider.min.js"></script>
+    <script type="text/javascript">
```
Comment (shakeelmohamed) [inline index.html]: What happened here?

File: index.html [hunk 7]
```diff
@@ -83,7 +90,18 @@
         <br>
         <!-- Twitter hashtag button: #ZenAudioPlayer -->
         <div id="tweetButton"></div>
-        <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
```
Comment (shakeelmohamed) [inline index.html]: Can we undo this?

File: index.html [hunk 8]
```diff
@@ -91,16 +109,17 @@
         <!-- Sponsor links -->
         <br>
         <h3>Sponsored by:</h3>
-        <p><a href="https://www.browserstack.com/" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack logo"></a></p>
```
Comment (shakeelmohamed) [inline index.html]: undo please

File: index.html [hunk 9]
```diff
@@ -91,16 +109,17 @@
...
+        <p>
+            <a href="https://www.browserstack.com/" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack Sponsored Image"></a>
```
Comment (shakeelmohamed) [inline index.html]: `"Browserstack logo" was fine, I don't think `"Browserstack Sponsored Image"` is necessary

Same goes for TrackJS

File: index.html [hunk 10]
```diff
@@ -91,16 +109,17 @@
...
+        </p>
+        <p>
+            <a href="https://trackjs.com/?utm_source=badges" target="_blank"><img src="img/trackjs.png" alt="TrackJS Sponsored Image"></a>
```
Comment (shakeelmohamed) [inline index.html]: Same as above, also let's rename the image to `img/trackjs_logo.png`

File: index.html [hunk 11]
```diff
@@ -91,16 +109,17 @@
...
+        </p>
     </footer>
     <!-- Schema.org structured data -->
     <script type="application/ld+json">
-        {
```
Comment (shakeelmohamed) [inline index.html]: Interesting, in other places the code was expanded, here it's collapsed - please revert

File: index.html [hunk 12]
```diff
@@ -91,16 +109,17 @@
...
-            "@context": "http://schema.org",
-            "@type": "Organization",
-            "url": "http://zen-audio-player.github.io",
-            "logo": "http://zen-audio-player.github.io/img/zen-audio-player-905.png"
-        }
+        { "@context": "http://schema.org", "@type": "Organization", "url": "http://zen-audio-player.github.io", "logo": "http://zen-audio-player.github.io/img/zen-audio-player-905.png" }
     </script>
 </body>
-</html>
```
Comment (shakeelmohamed) [inline index.html]: please revert

File: index.html [hunk 13]
```diff
@@ -1,27 +1,35 @@
...
     <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
     <link rel="icon" href="img/favicon.ico" type="image/x-icon">
-
     <link rel="stylesheet" href="bower_components/primer-css/css/primer.css">
     <link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.min.css">
     <link rel="stylesheet" href="bower_components/seiyria-bootstrap-slider/dist/css/bootstrap-slider.min.css">
     <link rel="stylesheet" href="css/styles.css">
-  
+    <script type="text/javascript">
+    window._trackJs = {
```
Comment (shakeelmohamed) [inline index.html]: nit: indent these 3 lines of code by 4 spaces

File: index.html [hunk 14]
```diff
@@ -1,27 +1,35 @@
 <!doctype html>
 <html lang="en">
+
 <head>
     <meta charset="utf-8">
     <meta name="description" content="Listen to YouTube videos, without the distracting visuals">
     <meta name="author" content="Shakeel Mohamed">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="google-site-verification" content="D3SjNR3tmNYOusESQijh_oH5SGmU9QsAIVwlqizwRBU" />
-
```
Comment (shakeelmohamed) [inline index.html]: Please revert

File: index.html [hunk 15]
```diff
@@ -1,27 +1,35 @@
...
     <title>Zen Audio Player</title>
-
```
Comment (shakeelmohamed) [inline index.html]: Please revert

File: index.html [hunk 16]
```diff
@@ -1,27 +1,35 @@
...
     <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
     <link rel="icon" href="img/favicon.ico" type="image/x-icon">
-
```
Comment (shakeelmohamed) [inline index.html]: Please revert

File: index.html [hunk 17]
```diff
@@ -1,27 +1,35 @@
 <!doctype html>
 <html lang="en">
+
```
Comment (shakeelmohamed) [inline index.html]: Please revert

File: index.html [hunk 18]
```diff
@@ -45,7 +53,6 @@
             </div>
         </form>
         <br>
-
```
Comment (shakeelmohamed) [inline index.html]: Please revert

**Comment by shakeelmohamed** [comment]
The diff got crazy with that last commit

**Comment by thomasjosephgreco** [comment]
@shakeelmohamed my fault i made so many commit! let me know that everything is alright though :smile:

File: index.html [hunk 19]
```diff
@@ -91,7 +92,8 @@
         <!-- Sponsor links -->
         <br>
         <h3>Sponsored by:</h3>
-        <p><a href="https://www.browserstack.com/" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack logo"></a></p>
+        <p><a href="http://bit.ly/1m2uotc" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack logo"></a></p>
+        <p><a href="https://trackjs.com/?utm_source=badges" target="_blank"><img src="img/trackjs.png" alt="Browserstack logo"></a></p>
```
Comment (shakeelmohamed) [inline index.html]: Alt text needs to be for TrackJS

File: index.html (hunk 8 above)
Comment (shakeelmohamed) [inline index.html]: This is still incorrect

**Comment by shakeelmohamed** [comment]
@thomasjosephgreco almost there!

File: index.html [hunk 20]
```diff
@@ -16,7 +16,8 @@
     <link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.min.css">
     <link rel="stylesheet" href="bower_components/seiyria-bootstrap-slider/dist/css/bootstrap-slider.min.css">
     <link rel="stylesheet" href="css/styles.css">
-  
+    <script type="text/javascript">window._trackJs = { token: 'f58d0b7c4b4248939fddc582f9e55c9d' };</script>
```
Comment (shakeelmohamed) [inline index.html]: Change these to double quotes please

**Comment by shakeelmohamed** [comment]
@thomasjosephgreco You are a rockstar! Thank you so much for bearing with me and tackling 2 birds with 1 PR!

**Pull request merged**